integrations/os2sync/os2mo.py

```diff
@@ -69,7 +69,8 @@ def os2mo_get(url, **params):
 def user_uuids(**kwargs):
     return [
         e["uuid"]
-        for e in os2mo_get("{BASE}/o/{ORG}/e/", limit=9999999, **kwargs).json()["items"]
+        for e in os2mo_get("{BASE}/o/{ORG}/e/", limit=9999999,
+                           **kwargs).json()["items"]
     ]
```

Comment:
En anden mulighed for disse, vha. `itemgetter` og `map`:
```
from operator import itemgetter
items = os2mo_get("{BASE}/o/{ORG}/e/", ...).json()["items"]
return map(itemgetter("uuid"), items)
```
Det har den bonus at map returnere en generator, så det faktisk kun koster noget, når resultatet bliver consumeret.